app-ios/TutanotaSharedFramework/Sql/SignalTokenizer.swift

```diff
@@ -0,0 +1,55 @@
+/**
+ * High-level Swift wrapper over the `signal_tokenizer` tokenizer.
+ */
+public class SignalTokenizer {
+	private var signalTokenizerHandle: UnsafeMutableRawPointer
+
+	// These signatures must be kept in sync!
+	private var signalTokenize: @convention(c) (UnsafePointer<Int8>) -> UnsafeMutableRawPointer
```

Comment:
should we even maybe wrap it in a framework https://forums.swift.org/t/using-a-dynamically-linked-c-library-dylib-from-spm/47839/8 :thinking: would still allow sqlcipher to load it, right?